pkg/converter/convert_test.go

```diff
@@ -11,6 +15,18 @@ import (
 )
 
 func TestConvert(t *testing.T) {
+	const _kc = "KUBECONFIG"
+	curEnvVal := os.Getenv(_kc)
+	tempCfg := fmt.Sprintf("%s/GoTests-%d.%s", os.TempDir(), time.Now().UnixNano(), _kc)
+	tempCfg = filepath.FromSlash(tempCfg)
+	os.Setenv(_kc, tempCfg)
+
+	defer func() {
+		// put it back the way it was and cleanup.
+		os.Setenv(_kc, curEnvVal)
```

Comment:
this is not needed as the env is set in this process and will go away after process exists

Reply:
Spot on! New PR Sent!